data/changelog.txt

```diff
@@ -147,438 +150,97 @@ Add reminder effects for some medicine.
-Bone flute
-Diesel V12 engine and Heavy Duty Cargo Truck
-Added metal and sandbag backstop to s_gun0.json gun range, and earplugs
-Crafting soap out of lye made from ash
-Micro Atomic Plant for Bright Nights mod
-Parks and recreation buildings
-Added pickles jar and recipe to make them from cucumbers
-More mixed alcohol recipes
-Add butcher shop
-Add ACOG scope
-Mansion Upgrade Project
-Add recipe for clean water using water purifier
-Added Hávamál and add changes to standing stones location
-Cash cards to plastic chunks recipe
-Initial work on multi-story houses
-Add magazine well for the uzi
-Welding goggles craft from tinted lenses
-Metalcraft: crown, ring, and bracelet
-Boil lye into powder
-Make football fields more likely to spawn.
-Make chainmail craftable from scratch
-Adds wasteland sausages
-Add bike shop
-Fix Anfo and RDX charge requirements.
-Add food material
-Update description of bone meal and chitin powder
-bullet pulling as quality
-Enable to forge a screwdriver
-Add reloaded 7.62x25mm to item group
-Create recipe for seed potatoes
-Add NPC mutation rounds.
-Update trucks.json
-Update vans_busses.json
-Potato companion
-Increase firehelmet weight.
-Update garage layout
-Disassembly of cardboard box
-Second variant of s_bikeshop
-Canned meat can be heated up
+Veterinarian Clinic.
+Implement disintegrating ammo linkages.
+NPC trade update.
+Creation of 5 new overmap special campsites.
+3 new roadside rest stops.
+Add magazine coloring, improve ammo/gun coloring.
+Prison Break Scenario.
+Funeral home.
+Razorclaws and Shipwreck.
+Add alternative triffid groves.
+Add 2x2 cemetery.
+Adds small Ponds.
+Adds Apple Orchard to the game.
+New characters: Start with loaded/holstered guns and sheathed blades.
+Add ability to steal items from NPC.
+Sugar House mapgen.
+Add 'Reach Refugee Center' mission.
+Add detergent and allow it to be used in washboard.
+Dairy farm.
+Micro Atomic Plant for Bright Nights mod.
+Parks and recreation buildings.
+Add butcher shop.
+Mansion Upgrade Project.
+Initial work on multi-story houses.
+Make chainmail craftable from scratch.
+Add bike shop.
 Add MShockXotto+ tileset
-Added restroom to s_arcade
-Added Tokarev pistol spawns
-Added rack of obscure ammo to gunsmith store
-adds descriptions to (not all) furniture objects
-Small town buildings
-Subway
-Glock-17 And Glock Drum Magazines
-Hallucination monsters are now described in extended description
-Added glazed tenderloin to house_patio
-Added PERMEABLE and "connects_to": "WALL", to rebar cage and set move_cost to 0
-Vehicle mounted washing machine
-Added vehicle-mounted washing machine deconstruction recipe
-Fungal zombie child
-Dissemble alternator with welder
-Added gigantic naked mole rat and acid ants
-Define custom butchery drop for broken cyborg
-Add .357 SIG ammunition and a P226
-New monster ability 'ABSORBS_SPLITS'
-New pain increasing mutations.
-Remove hardcoded mansions
-Add dashboards to vehicles
```

Comment:
This should be mentioned if it's not in some other section.

Reply:
It is now, and expanded a bit.